pkg/scaffold/cmd.go

```diff
@@ -43,6 +43,9 @@ import (
 	"os"
 	"runtime"
 
+	// Import all Kubernetes client auth plugins, e.g. Azure, GCP, OIDC, etc.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
@joelanford Do you think we could confirm this with a manual against GKE?

I've always seen each of the plugins imported separately
```Go
import (
    _ "k8s.io/client-go/plugin/pkg/client/auth/gcp
    _ "k8s.io/client-go/plugin/pkg/client/auth/azure
)
```
Just want to confirm if this method of importing all plugins works as well.

Reply:
[It should work](https://github.com/kubernetes/client-go/blob/master/plugin/pkg/client/auth/plugins.go), but we should test it to double check and to make sure the imports are in the correct spots.